trace-dispatcher/src/Cardano/Logging/Types.hs

```diff
@@ -52,8 +52,8 @@ import qualified Data.Aeson as AE
 import qualified Data.Aeson.Text as AE
 import qualified Data.HashMap.Strict as HM
 import           Data.IORef
-import           Data.Map (Map)
-import qualified Data.Map as Map
+import           Data.Map.Strict (Map)
+import qualified Data.Map.Strict as Map
 import qualified Data.Map.Strict as SMap
```

Comment:
We are importing `Data.Map.Strict` twice here.  Also wondering if we should import `as SMap` to prevent confusion.

Reply:
I would actually just suggest replacing all uses of `SMap` with `Map`.

Reply:
So `LMap` if we ever need to reach for a lazy map?

Reply:
If it ever comes up, personally, I'd suggest importing `Data.Map.Lazy` as `Lazy` unless that clashes with something, because it reads rather nicely.
The underlying datatype is shared between `Data.Map.Lazy` and `Data.Map.Strict`, so simply using `Data.Map.Strict` isn't a magic bullet, since something innocuous like `fmap` won't keep your values in WHNF and you can easily be given "lazy" maps where the values aren't already in WHNF. Honestly, I think it'd be better off as a newtype if you "really" want strictness, but that's probably be a discussion for another time.

Reply:
I think `Lazy` is too generic (both `Text` and `ByetString` have lazy versions. I would name is `LazyMap`.

Reply:
I'm changing it to `Map` in this PR. However John and Robert can have the ultimate decision (whilst considering Erik's input) and open a new PR with the relevant changes. This PR should also update the style doc so we can compile a list of how modules should be imported so it's not ambiguous.